app/metal-controller-manager/api/v1alpha1/environment_types.go

```diff
@@ -65,6 +71,42 @@ type EnvironmentList struct {
 	Items           []Environment `json:"items"`
 }
 
+// EnvironmentDefaultSpec returns EnvironmentDefault's spec.
+func EnvironmentDefaultSpec(talosRelease, endpoint string) *EnvironmentSpec {
+	return &EnvironmentSpec{
+		Kernel: Kernel{
+			Asset: Asset{
+				URL: fmt.Sprintf("https://github.com/talos-systems/talos/releases/download/%s/vmlinuz-%s", talosRelease, runtime.GOARCH),
+			},
+			Args: []string{
+				"console=tty0",
+				"console=ttyS1,115200n8",
+				"consoleblank=0",
+				"earlyprintk=ttyS1,115200n8",
+				"ima_appraise=fix",
+				"ima_hash=sha512",
```

Comment:
Major part of these args can be sourced from https://github.com/talos-systems/talos/blob/master/pkg/machinery/kernel/kernel.go